R/staggregate.R

```diff
@@ -1,25 +1,37 @@
 # Function to convert raster to data.table from https://gist.github.com/etiennebr/9515738
-as.data.table.raster <- function(x, row.names = NULL, optional = FALSE, xy=FALSE, inmem = raster::canProcessInMemory(x, 2), ...) {
+as_data_table_terra <- function(x, row.names = NULL, optional = FALSE, xy=FALSE, inmem = terra::inMemory(x), ...) {
```

Comment:
Better documentation could be nice

Reply:
Will do. This isn't a user level function but I agree there should be some explanation for someone reading the code.